source/users/custom-metadata.txt

```diff
@@ -296,3 +301,227 @@ Access Custom User Data from a Client Application
+   .. tab::
+      :tabid: cli
+      
+      .. procedure::
+
+         .. step:: Pull the Latest Version of Your App
+
+            To update your app with {+cli-bin+}, you need a local copy of its
+            configuration files.
+
+            To pull a local copy of the latest version of your app, run the following:
+
+            .. code-block:: bash
+
+               realm-cli pull --remote="<Your App ID>"
+
+            .. tip::
+
+               You can also download a copy of your application's configuration files from
+               the App Services UI. Go to the :guilabel:`Deploy > Export App` screen from
+               the App Dashboard.
+
+
+         .. step:: Configure User Metadata
+
+            You can find authentication provider :ref:`metadata_fields
+            <config-auth-providers>` for your app in ``/auth/providers.json``.
+            Update this array to request user metadata from the authentication provider.
+
+            **Google or Facebook**
+
+            This array resembles:
+
+            .. code-block:: javascript
+               :caption: /auth/providers.json
+
+               { ...other config details...
+                 "metadata_fields": [
+                   {
+                     "required": false,
+                     "name": "name"
+                   },
+                   {
+                     "required": false,
+                     "name": "gender"
+                   }
+                 ]
+               }
+
+            **Custom JWT authentication**
+
+            The metadata_fields array has an additional property, ``field_name``.
+            In custom JWT authentication, ``name`` represents the path to the field.
+            The ``field_name`` represents the name of the field.
+
+            .. code-block:: javascript
+               :caption: /auth/providers.json
+
+               { ...other config details...
+                 "metadata_fields": [
+                   {
+                     "required": true,
+                     "name": "user.name",
+                     "field_name": "name"
+                   },
+                   {
+                     "required": false,
+                     "name": "user.favoriteColor",
+                     "field_name": "favoriteColor"
+                   }
+                 ]
+               }
+
+
+         .. step:: Deploy the Custom User Data Configuration
+
+            Once you've configured custom user data, you can push the updated config to
+            your remote app. {+cli+} immediately deploys the update on push.
+
+            .. code-block:: bash
+
+               realm-cli push --remote="<Your App ID>"
+
+
+
+Access User Metadata from a Client Application
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. tabs-realm-sdks::
+
+   .. tab::
+      :tabid: ios
+
+      .. tip:: Read a User's Metadata
+         
+         Learn how to :ref:`read a user's metadata from an iOS
+         application <ios-read-user-metadata>`.
+
```

Comment:
[suggestion] pretty sure other SDKs have this info. worth looking into

Reply:
Thanks! Added this suggestion but could only find info for Flutter and .NET.

Reply:
that's all the docs that prob exist for it now. was added as part of new IA updates.

bonus points if you add tickets for adding to other SDKs and then add links to those docs from this page.

Reply:
thanks for making those david! and i think merging this before those is OK. tho noting in those tickets to also update here would make sense.

Reply:
Sounds good! Just added a note in the tickets and will merge this in soon.